src/dto/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .ModuleInfo import ModuleInfo
```

Comment:
```suggestion
from . import ModuleInfo.ModuleInfo
```

Reply:
```
Traceback (most recent call last):
  File "/Users/til.pockrandt/projects/JustForYou/src/main.py", line 10, in <module>
    import dto
  File "/Users/til.pockrandt/projects/JustForYou/src/dto/__init__.py", line 1
    from . import ModuleInfo.ModuleInfo
                                            ^
SyntaxError: invalid syntax
```

Reply:
```suggestion
from . import ModuleInfo
```

ich hab die import gerne so hier, damit sieht von wo die sachen alle kommen

Reply:
Können wir so machen. Das macht aber auch, dass wir dann `from dto.ModuleInfo import ModuleInfo` verwenden müssen, um die Klasse zu importieren. Das finde ich nicht so schön.